.github/workflows/ci.yml

```diff
@@ -31,7 +31,6 @@ jobs:
     - name: Build OS packages
       run: .github/scripts/generate-os-packages.sh
       shell: bash
-      if: runner.os == 'Windows'
```

Comment:
In last week I want to generate os packages for linux and test them, but I was blocked, because the stage `CI / native-tests (ubuntu-latest) ` was failing.  So I removed this line to run os package generation on `jvm-tests (ubuntu-latest)`.

And then I forgot to restore this line, I will do it in next commit.